### PR TITLE
Replicate comments revival

### PR DESCRIPTION
**Pull request by dickolsson**

### CODE REVIEW

File: src/Normalizer/ContentEntityNormalizer.php [hunk 1]
```diff
@@ -251,7 +263,10 @@ public function denormalize($data, $class, $format = NULL, array $context = arra
           );
           $file = \Drupal::getContainer()->get('serializer')->deserialize($value['data'], '\Drupal\file\FileInterface', 'base64_stream', $file_context);
           if ($file instanceof FileInterface) {
-            $data[$field_name][$delta] = array('entity_to_save' => $file);
+            $data[$field_name][$delta] = array(
```
Comment (timmillwood) [inline src/Normalizer/ContentEntityNormalizer.php]: Let's use `[]` instead of `array()`.

File: src/Normalizer/EntityReferenceItemNormalizer.php [hunk 2]
```diff
@@ -47,10 +47,18 @@ public function normalize($field, $format = NULL, array $context = array()) {
       return $value;
     }
 
-    return array(
+    $field_info = array(
```
Comment (timmillwood) [inline src/Normalizer/EntityReferenceItemNormalizer.php]: `[]`

File: src/Plugin/rest/resource/DbResource.php [hunk 3]
```diff
@@ -143,47 +134,20 @@ public function post($workspace, ContentEntityInterface $entity = NULL) {
...
-    // entities with the correct values.
-    \Drupal::service('relaxed.stub_entity_processor')->processEntity($entity);
-
     // Validate the received data before saving.
     $this->validate($entity);
     try {
       $entity->save();
       $rev = $entity->_rev->value;
-      $response = new ResourceResponse(array('ok' => TRUE, 'id' => $uuid, 'rev' => $rev), 201, array('ETag' => $rev));
+      $response = new ResourceResponse(array('ok' => TRUE, 'id' => $entity->uuid(), 'rev' => $rev), 201, array('ETag' => $rev));
```
Comment (timmillwood) [inline src/Plugin/rest/resource/DbResource.php]: `[]`

File: relaxed.install [hunk 4]
```diff
@@ -1,15 +1,35 @@
 <?php
 
+use Drupal\user\Entity\Role;
+
 /**
  * Implements hook_install().
  */
 function relaxed_install() {
+  // Demonstrate best security practise and create a separate role with the
```
Comment (jeqq) [inline relaxed.install]: I think here is a typo - practise => practice.

File: src/Tests/StubTest.php [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+namespace Drupal\relaxed\Tests;
+
+use Drupal\field\Tests\EntityReference\EntityReferenceTestTrait;
+use Drupal\multiversion\Entity\Workspace;
+use Drupal\simpletest\WebTestBase;
+
+/**
+ * @group multiversion
+ *
+ * @todo It's too late at night and I can't figure out how to get this test to
```
Comment (timmillwood) [inline src/Tests/StubTest.php:12]: Have you slept yet?

Comment (dickolsson) [inline src/Tests/StubTest.php:12]: Haha, not yet. So I'll leave that in there for now :wink:

**Pull request merged**